src/materialized/tests/cli.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright Materialize, Inc. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use std::ffi::OsStr;
+use std::os::unix::ffi::OsStrExt;
+use std::time::Duration;
+
+use assert_cmd::Command;
+use predicates::prelude::*;
+
+fn cmd() -> Command {
+    let mut cmd = Command::cargo_bin("materialized").unwrap();
+    cmd.env_clear()
+        .env("MZ_DEV", "1")
+        .timeout(Duration::from_secs(10));
+    cmd
+}
+
+#[test]
+fn test_threads() {
```

Comment:
can you also test with -1 just in case?

Reply:
It's really hard to do that :(. There's a note at the bottom of the test file. Happy to entertain suggestions on how to test that, if you've got any. Possibly we should stamp a file when server startup is successful.

Reply:
I meant test that it fails with negative one

Reply:
ohh yeah good idea

Reply:
Done.